webapp/channels/src/components/help/formatting.tsx

```diff
@@ -0,0 +1,719 @@
+                    <h2>
+                        <FormattedMessage
+                            id='help.formatting.style.title'
+                            defaultMessage='Text Style'
+                        />
+                    </h2>
+                    <p>
+                        <FormattedMessage
+                            id='help.formatting.style.description'
+                            defaultMessage='You can use either _ or * around a word to make it italic. Use two to make a word bold.'
+                        />
+                    </p>
+                    <table className='Help__table'>
+                        <thead>
+                            <tr>
+                                <th>
+                                    <FormattedMessage
+                                        id='help.formatting.table.text_entered'
+                                        defaultMessage='Text Entered'
+                                    />
+                                </th>
+                                <th>
+                                    <FormattedMessage
+                                        id='help.formatting.table.how_it_appears'
+                                        defaultMessage='How it appears'
+                                    />
+                                </th>
+                            </tr>
+                        </thead>
+                        <tbody>
+                            <tr>
+                                <td><code>{'_italic_'}</code></td>
+                                <td><em>{'italic'}</em></td>
+                            </tr>
+                            <tr>
+                                <td><code>{'**bold**'}</code></td>
+                                <td><strong>{'bold'}</strong></td>
+                            </tr>
+                            <tr>
+                                <td><code>{'~~strikethrough~~'}</code></td>
+                                <td><del>{'strikethrough'}</del></td>
+                            </tr>
+                            <tr>
+                                <td><code>{'**_bold-italic_**'}</code></td>
+                                <td><strong><em>{'bold-italic'}</em></strong></td>
+                            </tr>
+                        </tbody>
+                    </table>
+                </section>
+
+                <section className='Help__section'>
+                    <h2>
+                        <FormattedMessage
+                            id='help.formatting.code_block.title'
+                            defaultMessage='Code Block'
+                        />
+                    </h2>
+                    <p>
+                        <FormattedMessage
+                            id='help.formatting.code_block.description'
+                            defaultMessage='Create a code block by indenting each line by four spaces, or by placing ``` on the line above and below your code.'
```

Comment:
```suggestion
                            defaultMessage='Create a code block by indenting each line by four spaces, or by placing <code>```</code> on the line above and below your code.'
```